ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/translator/SqlFunctionConverter.java

```diff
@@ -392,10 +392,11 @@ private static String getName(GenericUDF hiveUDF) {
       registerFunction("istrue", SqlStdOperatorTable.IS_TRUE, hToken(HiveParser.Identifier, "istrue"));
       registerFunction("isnotfalse", SqlStdOperatorTable.IS_NOT_FALSE, hToken(HiveParser.Identifier, "isnotfalse"));
       registerFunction("isfalse", SqlStdOperatorTable.IS_FALSE, hToken(HiveParser.Identifier, "isfalse"));
-      registerFunction("is not distinct from", SqlStdOperatorTable.IS_NOT_DISTINCT_FROM, hToken(HiveParser.EQUAL_NS, "<=>"));
```

Comment:
Just to confirm, does this mean that `is not distinct from` was never used?

Reply:
the calcite2hive translation was not working - and thrown an exception
I'm not sure about hive2calcite ; I've made a note on HIVE-23594 to check out what was happening